cmd/srcd/cmd/components.go

```diff
@@ -43,7 +43,7 @@ var componentsListCmd = &cobra.Command{
 
 		cmps, err := components.List(context.Background(), allVersions)
 		if err != nil {
-			return fatal(err, "could not list images")
+			return wrapHumanizeErr(err, "could not list images")
```

Comment:
nit-pick: because we use this function a lot I would prefer shorter name. Options from the top of my head:
- `humanizef`
- `wrapf`
- `userErr`

Reply:
I don't know maybe `whumanizef`? so that we have both `humanize` and `whumanizef`.

Reply:
> whumanizef

Please don't :smile:

Reply:
😂 I think that `humanizef` is enough.

Reply:
Addressed [here](https://github.com/src-d/engine/pull/359/commits/bc9a7252f9cc117bbb0fdd56057ca6977d2f137d).